src/main/python/hydra/helloworldtest/runtest.py

```diff
@@ -0,0 +1,84 @@
+import time
+
+from hydra.lib.runtestbase import RunTestBase
+try:
+    # Python 2.x
+    from ConfigParser import ConfigParser
+except ImportError:
+    # Python 3.x
+    from configparser import ConfigParser
+
+
+class HW(RunTestBase):
+    def __init__(self):
+        self.config = ConfigParser()  # Object contains 'configurations' parsed from hydra.ini config file
+        RunTestBase.__init__(self, 'HelloWorld', None, None, startappserver=True, mock=False)
```

Comment:
If defaults for mock is false and startappserver is true then those should be omitted.